packages/create-plugin/templates/datasource/provisioning/datasources/datasources.yml

```
@@ -0,0 +1,10 @@
+apiVersion: 1
+
+datasources:
+  - name: '{{ pluginName }}'
+  - type: '{{ normalize_id pluginName orgName 'datasource' }}'
```

Scaffolding a datasource plugin results in the docker compose bailing with:

```
heywesty-myplugin-datasource  | logger=provisioning t=2023-11-23T15:13:33.486031168Z level=error msg="Failed to provision data sources" error="Datasource provisioning error: unique identifier and org id are needed to be able to get or delete a datasource"
heywesty-myplugin-datasource  | Error: ✗ Datasource provisioning error: unique identifier and org id are needed to be able to get or delete a datasource
heywesty-myplugin-datasource exited with code 1
```

It looks like it's caused by this rogue `-`.

```suggestion
    type: '{{ normalize_id pluginName orgName 'datasource' }}'
```